config/prowgen/pkg/context.go

```diff
@@ -156,19 +210,6 @@ func (pc *ProwContext) periodicDashboardName() string {
 	return fmt.Sprintf("%s-periodics-%s", pc.Repo, pc.Branch)
 }
 
-func (pc *ProwContext) cronSchedule(periodicityHours int) string {
-	minute := pc.minutesValue()
-
-	return fmt.Sprintf("*/%d %d * * *", minute, periodicityHours)
-}
-
-// minutesValue returns a minute value (0 - 59) at which a test should be run and then
-// increases the next value returned. This helps to prevent every test running at the same
-// minute within the hour causing a spiky distribution of tests.
-func (pc *ProwContext) minutesValue() int {
-	minuteVal := pc.minutesCounter.Minute()
-
-	pc.minutesCounter = pc.minutesCounter.Add(4 * time.Minute)
-
-	return minuteVal
+func cronSchedule(minute int, startHour int, periodicityHours int) string {
+	return fmt.Sprintf("%02d %02d-23/%02d * * *", minute, startHour, periodicityHours)
```

Comment:
For my information: how can the following be expressed?
A job that should run every 5 hours starting from hour 1.
eg.:
01:00:00
06:00:00
11:00:00
16:00:00
21:00:00

02:00:00
07:00:00
12:00:00
17:00:00
22:00:00

03:00:00
08:00:00
13:00:00
18:00:00
23:00:00

...

Reply:
This is interesting: https://stackoverflow.com/a/70618521
Seems like the only simple solution is to have the cron run at 0, 5, 10, 15, 20 with 4 hours between the last hour and the next day.

So the following expressions would resolve to the following sequences:
0-23/5 -> 0, 5, 10, 15, 20 (equivalent to */5)
1-23/5 -> 1, 6, 11, 16, 21
2-23/5 -> 2, 7, 12, 17, 22
3-23/5 -> 3, 8, 13, 18, 23
Next sequence == 0-23/5.